lib/cooked_post_processor.rb

```diff
@@ -285,10 +285,11 @@ def create_span_node(klass, content=nil)
     span
   end
 
-  def update_topic_image
-    if @post.is_first_post?
-      img = extract_images_for_topic.first
-      @post.topic.update_column(:image_url, img["src"][0...255]) if img["src"].present?
+  def update_post_image
+    img = extract_images_for_post.first
```

Comment:
The `if img["src"].present?` condition is required to verify that image is present in topic/post before updating `image_url` field.

Also, that condition was already present in previous code for topics, I was just making sure that it applied to posts as well 😄